src/NATS.Client/Internals/JsonUtils.cs

```diff
@@ -73,5 +74,10 @@ internal static JSONArray ToArray(List<string> list)
             }
             return arr;
         }
+ 
+        public static byte[] SimpleMessageBody(string name, object value)
+        {
+            return Encoding.ASCII.GetBytes("{\"" + name + "\":" + value.ToString() + "}");
```

Comment:
Also, will we need to consider escaping of `value` here, or is that on the caller?
```suggestion
            return Encoding.UTF8.GetBytes("{\"" + name + "\":" + value.ToString() + "}");
```

Reply:
right now the only use is where the object is a number. Also internal uses should never have anything other than ascii

Reply:
Suggested as JSON is UTF8 encoded. I didn't suggest interpolation due to the braces, otherwise the rules for escaping quotes are based on whether or not the string is literal or not (i.e. prefixed with `@`):
```suggestion
            return Encoding.UTF8.GetBytes($"{{\"{name}\":{value}}}");
```
An alternative using an interpolated literal string:
```csharp
            return Encoding.UTF8.GetBytes($@"{{""{name}"":{value}}}");
```